Externals/discord-rpc/src/discord-rpc.vcxproj

```diff
@@ -53,6 +61,24 @@
     </Midl>
     <Lib />
   </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM64'">
```

Comment:
It looks like the changes to this file can also be reduced by just removing the `$(Platform)` component of the existing `ItemDefinitionGroup` for x64.

Reply:
+1, doesn't seem like any of this is ARM64 specific.